pkg/analyze/common_status.go

```diff
@@ -64,7 +65,7 @@ func commonStatus(outcomes []*troubleshootv1beta2.Outcome, name string, iconKey
 			if exists == false && outcome.Warn.When != "absent" {
 				result.IsFail = true
 				result.Message = fmt.Sprintf("The %s %q was not found", resourceType, name)
-				result.URI = outcome.Fail.URI
```

Comment:
see https://github.com/replicatedhq/troubleshoot/pull/1333/files#diff-d2f5edb6b9cc4705fc281663c6648c1792d04ad3c4e7e732167a96011dbca51cL107 similar thoughts but for `outcome.Warn`